### PR TITLE
feat: user login, permissions and profile

### LINKED ISSUE #255: Permission handling
The sidekick should be able to handle 401 and 403 from the admin API.

Reference: https://github.com/adobe/helix-admin/issues/460

### PR DESCRIPTION
**Pull request by rofe**
Fix #255 

Testing: Options > Advanced > Development > Use specific branch version: `issue-255`

### CODE REVIEW

**Comment by rofe** [comment]
States: 
1. Not logged in:
   image
2. Not logged in, project requires login:
   image
3. Logged in:
   image

File: test/app.test.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ describe('Test sidekick bookmarklet', () => {
     assert.ok(!sidekick, 'Did render with missing config');
   }).timeout(IT_DEFAULT_TIMEOUT);
 
-  it('Renders with config', async () => {
+  it.only('Renders with config', async () => {
```
Comment (kptdobe) [inline test/app.test.js:37]:
```suggestion
  it('Renders with config', async () => {
```

**Review by kptdobe — APPROVED** [review]

**Review by tripodsan — APPROVED** [review]

**Comment by rofe** [comment]
@kptdobe could you provide the French translations for this feature? You can commit directly to the `issue-255` branch.

01ef59d6df0cb2da9a88685efbc8eab74cda5859

(I also noticed a few others that were missing in French... 😬)

**Review by davidnuescheler — APPROVED** [review]

**Pull request merged**